### PR TITLE
chore: add tags,enabled,archived to update feature api

### PR DESCRIPTION
**Pull request by kentakozuka**
part of https://github.com/bucketeer-io/bucketeer/issues/904

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -26,7 +26,7 @@ LDFLAGS_BUILDDATE := $(LDFLAGS_PACKAGE).BuildDate
 #############################
 
 .PHONY: generate-all
-generate: proto-all mockgen
+generate-all: proto-all mockgen
```
Comment (kentakozuka) [inline Makefile:29]: fix

File: pkg/feature/domain/feature.go [hunk 2]
```diff
@@ -953,6 +962,38 @@ func (f *Feature) Update(
...
+				return nil, err
+			}
+			incVersion = true
+		} else {
+			if err := updated.Disable(); err != nil {
+				return nil, err
+			}
+			incVersion = true
+		}
+	}
```
Comment (cre8ivejp) [inline pkg/feature/domain/feature.go:983]:
```suggestion
	if enabled != nil {
		if enabled.Value {
			if err := updated.Enable(); err != nil {
				return nil, err
			}
		} else {
			if err := updated.Disable(); err != nil {
				return nil, err
			}
		}
		incVersion = true
	}
```

File: pkg/feature/domain/feature.go [hunk 3]
```diff
@@ -953,6 +962,38 @@ func (f *Feature) Update(
...
+				return nil, err
+			}
+			incVersion = true
+		} else {
+			if err := updated.Unarchive(); err != nil {
+				return nil, err
+			}
+			incVersion = true
+		}
+	}
```
Comment (cre8ivejp) [inline pkg/feature/domain/feature.go:996]:
```suggestion
	if archived != nil {
		if archived.Value {
			if err := updated.Archive(); err != nil {
				return nil, err
			}
		} else {
			if err := updated.Unarchive(); err != nil {
				return nil, err
			}
		}
		incVersion = true
	}
```

**Review by cre8ivejp — COMMENTED** [review]
I  left a few comments. The rest looks good!

**Comment by kentakozuka** [comment]
@cre8ivejp Thanks!
Fixed 👍

**Review by cre8ivejp — APPROVED** [review]
Thank you!

**Pull request merged**